roles/teleport/tasks/config.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Ensure /etc/teleport directory exists
+  file:
```

Comment:
try to use FQCN in place of using module
like use  `ansible.builtin.file` in stead of `file`